lib/animina/accounts/resources/photo.ex

```diff
@@ -175,4 +195,75 @@ defmodule Animina.Accounts.Photo do
       reference :user, on_delete: :delete
     end
   end
+
+  def create_optimized_photos(record) do
+    case check_if_image_magick_is_installed() do
+      {:ok, _} ->
+        resize_images_with_imagemagick(record)
+
+      {:error, _} ->
+        copy_images_directly(record)
+    end
+  end
+
+  defp copy_images_directly(record) do
```

Comment:
Don't just copy the filename. Copy the file and create a new one. Do it as if you were using imagemagick. The result will be a new file which needs to be stored.

Reply:
Done , Thanks